test/sample-test/configs/xgboost_training_cm.config.yaml

```diff
@@ -16,9 +16,6 @@ test_name: xgboost_training_cm
 arguments:
   output:
   project: ml-pipeline-test
```

Comment:
I'll suggest setting `diagnostic_mode: false` so that we always check the downstream component in sample test. WDYT?

Reply:
That makes sense! updating.